build.sbt

```diff
@@ -965,23 +967,34 @@ lazy val awsTerraform: ProjectMatrix = (projectMatrix in file("serverless/aws/te
     )
   )
   .jvmPlatform(scalaVersions = scala2Versions)
+  .jsPlatform(scalaVersions = scala2Versions)
```

Comment:
do we need terraform and sam in the js versions?

Reply:
it seems like if we want to build a subproject for js, all of its dependencies must also define `jsPlatform`
```
no rows were found in awsTerraform matching ProjectRow(true, List(PlatformAxis(js,JS,js), ScalaVersionAxis(2.12.15,2.12))): List(ProjectRow(true, List(PlatformAxis(jvm,JVM,jvm), ScalaVersionAxis(2.12.15,2.12))), ProjectRow(true, List(PlatformAxis(jvm,JVM,jvm), ScalaVersionAxis(2.13.6,2.13))))
```

Reply:
ah and the example depend on terraform ... should they, though?